### PR TITLE
D8CORE-5257: working on footer links

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- changed out the link colors in D8CORE-5274
- changed the lockup to not have a default word mark

# Review By (Date)
- 2/11

# Criticality
- High
- Only this subtheme

# Urgency
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. update the lockup on the header and the footer.
4. Remove the standard lockup and add your own or have none
4. Verify it looks ok.
5. Review the link color ticket: https://stanfordits.atlassian.net/browse/D8CORE-5274
6. Create scenarios that match and don't match what they want.
7. Verify it works. 

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Minimally branded subtheme

# Associated Issues and/or People
- [D8CORE-5257](https://stanfordits.atlassian.net/browse/D8CORE-5257)
- [D8CORE-5274](https://stanfordits.atlassian.net/browse/D8CORE-5274)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by jdwjdwjdw — CHANGES_REQUESTED** [review]
Lockup updates looking good.

I created a test page for the link color updates for teasers and lists, and noticed a couple of areas where the cardinal red is still being applied:
- Publication Teasers - currently the Publication title stays black on hover, and link icon / taxonomy are cardinal red
Screen Shot 2022-02-09 at 11 01 55 AM
- News Default List - link icon and taxonomy are cardinal red. News Card List looks correct, so just the default list.
Screen Shot 2022-02-09 at 10 53 39 AM

**Comment by jenbreese** [comment]
@jdwjdwjdw Thank you.  I can't reproduce the news list one. Maybe we can chat?
Screen Shot 2022-02-10 at 8 30 01 AM

**Comment by jenbreese** [comment]
Hi @pookmish and @imonroe Here is the pr for the minimally branded subtheme. So you can take a look since it is an important subtheme too.  Thank you!

**Review by jdwjdwjdw — APPROVED** [review]
Looks like the issues I was seeing with the News default list was due to my News module being slightly out-of-date. Should be good now, as far as I can tell.

File: src/scss/components/local-footer/_local-footer.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+.su-local-footer__action-links,
+.su-local-footer__primary-links,
+.su-local-footer__secondary-links {
+  a {
+    display: block;
+    margin-bottom: 7px;
+  }
+}
+
+.su-local-footer__social-links {
```
Comment (imonroe) [inline src/scss/components/local-footer/_local-footer.scss]: should this be an empty class?

Comment (jenbreese) [reply src/scss/components/local-footer/_local-footer.scss]: Nope, it was placeholder that wasn't needed.

**Review by jdwjdwjdw — APPROVED** [review]
Looks great!

**Pull request merged**